syntax = "proto3";


package hypernets.dispatchers.predict.grpc.proto;


service PredictService {
  rpc predict(PredictRequest) returns (PredictResponse) {}
}

message PredictRequest {
  string data_file = 1;
  string result_file = 2;
}

message PredictResponse {
  string data_file = 1;
  string result_file = 2;
  int32  code = 3;
  string message = 4;
}
